src/main/java/br/com/vivo/model/Cliente.java

```diff
@@ -0,0 +1,39 @@
+package br.com.vivo.model;
+
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "cliente")
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
```

Comment:
coloca essa anotação -> @EqualsAndHashCode(of = "{id}")

Reply:
Feito!